protos/generate.sh

```diff
@@ -2,33 +2,42 @@
 set -e
 
 if [[ -z "$BENTOML_REPO" ]]; then
-  # Assuming running this script from anywhere within the BentoML git repository
-  BENTOML_REPO=$(git rev-parse --show-toplevel)
+    # Assuming running this script from anywhere within the BentoML git repository
+    BENTOML_REPO=$(git rev-parse --show-toplevel)
 fi
 
 PROTO_PATH=$BENTOML_REPO/protos
 PY_OUT_PATH=$BENTOML_REPO/bentoml/yatai/proto
+# test YataiService Interceptor calls
+# add different directory for tests proto for yatai
+PROTO_TEST_PATH=$PROTO_PATH/tests
```

Comment:
Could we move test related `.proto` files and generated `.py` files all under `./tests/` directory?  It's not ideal to include testing code in BentoML's final package distribution

Reply:
I had changed accordingly